src/controllers/light-the-bridge.js

```diff
@@ -23,36 +32,45 @@ module.exports = async function(req, res) {
     })
   }
 
-  console.log(`${tweet.title}`)
-  tweet = {
-    ...tweet,
-    ...{
+  // the 'data' object has fields: colour_s, reason_occasion, row_id
+  let data = queryResult[0]
+  let responseValues
+  let key = `light_the_bridge`
+  let latest = await req.cache.getLatest(key)
+  if (!data || (latest && latest.row_id == data.row_id)) {
+    console.log('No new color data.')
+    responseValues = await req.cache.getAll(key, req.body['limit'])
+  } else {
+    console.log(data.reason_occasion)
+    let id = uuid()
+    let colors = parseColors(data.colour_s)
+    let firstColor = colors[0] ? colors[0].hex : ''
+    let newData = {
+      id,
+      row_id: data.row_id,
+      title: data.reason_occasion,
```

Comment:
Since we are no longer getting our data from tweets, it might be better to just keep the `description` field and omit the `title` field. The `description` field can be populated by the `Reason/Occasion` column of the dataset. However, this might break some functionality with the existing trigger. Any thoughts on this?

Reply:
I think we should maintain functionality, even if it means mapping `Reason/Occasion` to both `title` and `description`.

Reply:
I agree. Removing one or the other might break some applets that may rely on them.